1.Core/UIElements/Drivers/GenericWebDriver.cs

```diff
@@ -8,6 +9,8 @@ public class GenericWebDriver
     public static IWebDriver Instance =>
         _driver = _driver == null ? WebDriverFactory.GetDriver(ConfigModel.DriverType) : _driver;
 
+    public static WebDriverWait Wait => new WebDriverWait(Instance, TimeSpan.FromSeconds(30));
```

Comment:
The value for the explicit wait (in this case 30 seconds) should be obtained from the ConfigBuilder

Reply:
Fixed